warehouse/client/client.go

```diff
@@ -55,6 +55,8 @@ func (cl *Client) sqlQuery(statement string) (result warehouseutils.QueryResult,
 			switch t := values[i].(type) {
 			case []uint8:
 				values[i] = string(t)
+			default:
```

Comment:
This would cause an issue I think for other datatypes. We added this only because in the later step we are doing `fmt.Sprintf("%+v", values[i])` which tends to print byte array which is difficult to read.
I would like to refactor it in this way.
```
       for rows.Next() {
		for i := 0; i < colCount; i++ {
			valuePtrs[i] = &values[i]
		}

		err = rows.Scan(valuePtrs...)
		if err != nil {
			return result, err
		}

		var stringRow []string
		for i := 0; i < colCount; i++ {
			switch t := values[i].(type) {
			case []uint8:
				stringRow = append(stringRow, fmt.Sprintf("%+v", string(t)))
			default:
				stringRow = append(stringRow, fmt.Sprintf("%+v", values[i]))
			}
		}
		result.Values = append(result.Values, stringRow)
	}
```

Reply:
Keeping an abstraction of different types of DB drivers is not a good way to have.